manifests/p/PKI/NCALayer/1.3/PKI.NCALayer.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NLSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: PKI.NCALayer
+PackageVersion: "1.3"
+PackageLocale: en-US
+Publisher: National Certification Authority of the Republic of Kazakhstan

```

```suggestion
Publisher: Национальный удостоверяющий центр РК
```
shows up as `Национальный удостоверяющий центр РК` on en-US devices.